tests/observability/runbook_url/test_runbook_url.py

```
@@ -1,13 +1,32 @@
 
 
+def validate_downstream_runbook_url(runbook_urls_from_prometheus_rule: dict[str, str]) -> None:
+    error_messages = {}
+    alerts_without_runbook = []
+
+    for alert_name, runbook_url in runbook_urls_from_prometheus_rule:
+        if not runbook_url:
+            LOGGER.error(f"For alert: {alert_name} Url not found")
+            alerts_without_runbook.append(alert_name)
+            continue
+        if requests.get(runbook_url, allow_redirects=False).status_code != http.HTTPStatus.OK:
```

Ruff (0.14.5)

Probable use of `requests` call without timeout

(S113)